lib/main_common.pm

```diff
@@ -515,6 +515,9 @@ sub load_extra_tests {
             loadtest "console/a2ps";    # a2ps is not a ring package and thus not available in staging
             loadtest "console/weechat";
             loadtest "console/nano";
+            if (get_var("IPSEC")) {
+                loadtest "console/ipsec_tools_h2h";
+            }
```

Comment:
So far this section has been kind of reserved for tests for packages that are not in SLE. At best we can really just add it to extra tests outside any if/else on DISTRI. Because it's depending on another test variable I don't see a problem with that. Then it should also not be added within products/sle/main.pm . Can you do that?

Reply:
I agree with you, I will move it out of if/else on DISTRI. But for SLE, this one is also used as a fips mult-machine case, I think I should keep it in products/sle/main.pm.